docs/pass-through.md

```diff
@@ -0,0 +1,237 @@
+# Running Antrea In Pass-Through Mode
```

Comment:
passthrough should be one word - so not pass-through, passThrough.
But I guess I need to understand the implementation better, to see if we should still call it passthrough.